### PR TITLE
Add --all (-a) flag to enable routes command (implements #360)

### LINKED ISSUE #360: Add `--all` (`-a`) flag to enable routes command
Add a flag to enable all available routes for a given node. Potentially also a flag/filter to enable all routes for all nodes?

### PR DESCRIPTION
**Pull request by samson4649**
- [x] read the [CONTRIBUTING guidelines](README.md#user-content-contributing)
- [x] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [x] added integration tests
- [ ] updated documentation if needed
- [ ] updated CHANGELOG.md

----------------------
Added cli functionality to enable all routes on host.

### CODE REVIEW

File: cmd/headscale/cli/routes.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ func init() {
 	enableRouteCmd.Flags().
 		StringSliceP("route", "r", []string{}, "List (or repeated flags) of routes to enable")
 	enableRouteCmd.Flags().Uint64P("identifier", "i", 0, "Node identifier (ID)")
+	enableRouteCmd.Flags().BoolP("all", "a", false, "All routes from host")
```
Comment (kradalby) [inline cmd/headscale/cli/routes.go:27]:
```suggestion
	enableRouteCmd.Flags().BoolP("all", "a", false, "Enable all routes on the given host")
```

**Review by kradalby — COMMENTED** [review]
Looks great, could you have a go at adding a integration test for it in `integration_cli_test.go` ?

**Comment by samson4649** [comment]
@juanfont - added integration test and fixed lint issues

**Review by juanfont — APPROVED** [review]

**Comment by juanfont** [comment]
@samson4649 looks awesome! Could you just add a small entry in the `CHANGELOG.md` ?

**Comment by samson4649** [comment]
@juanfont - change log updated

**Review by kradalby — APPROVED** [review]

**Pull request merged**